content-src/components/DiscoveryStreamComponents/DSCard/DSCard.jsx

```diff
@@ -174,7 +178,7 @@ export class DSCard extends React.PureComponent {
               sponsor={this.props.sponsor}
             />
           )}
-          {!this.props.cta_variant && (
+          {!isButtonCTA && (
```

Comment:
Could just use a ternary instead of 2 subsequent conditionals. See: https://reactjs.org/docs/conditional-rendering.html#inline-if-else-with-conditional-operator

Reply:
Done